bundle/jsky.app.ot/src/main/java/jsky/app/ot/gemini/editor/targetComponent/EdCompTargetList.java

```diff
@@ -118,22 +118,25 @@ public void textBoxAction(TextBoxWidget tbwe) {
     }
 
     @Override protected void updateEnabledState(boolean enabled) {
-        super.updateEnabledState(enabled);
+        if (enabled != isEnabled()) {
+            setEnabled(enabled);
+            updateEnabledState(getWindow().getComponents(), enabled);
+
+            // Update enabled state for all detail widgets.  The current editor
+            // will have already been updated by the super.updateEnabledState so
+            // update the others.
+            for (final TargetDetailEditor ed : _w.detailEditor.allEditorsJava()) {
+                if (_w.detailEditor.curDetailEdiorJava().forall(cur -> cur != ed)) {
```

Comment:
Thanks, I fixed the comment and the misspelling of "editor".  I didn't filter because it is a Java collection.  It looks like it can be easily converted to a Java 8 `Stream` and then filtered

````java
_w.detailEditor.allEditorsJava().stream().filter(ed -> _w.detailEditor.curDetailEditorJava().forall(cur != ed))
````
or something ... but I think I'll be conservative since we're nearing the release date.